sklearnex/dispatcher.py

```diff
@@ -289,6 +292,14 @@ def patch_sklearn(name=None, verbose=True, global_patch=False, preview=False):
             "for scikit-learn >= 0.22 only ..."
         )
 
+    if not no_msg and "sklearn" in sys.modules.keys():
```

Comment:
If name is specified, then the user is patching a specific estimator or function. Either the message should apply to only the function/estimator being patched, or the message should not be displayed.

Reply:
Very good point. I didn't think at all about "non-default" scenarios so far. Are there other "exotic" use cases that you can think of?